src/assets/i18n/en.json5

```diff
@@ -542,28 +542,45 @@
 
   "admin.metadata-import.breadcrumbs": "Import Metadata",
 
+  "admin.batch-import.breadcrumbs": "Import Batch",
+
   "admin.metadata-import.title": "Import Metadata",
 
+  "admin.batch-import.title": "Import Batch",
+
   "admin.metadata-import.page.header": "Import Metadata",
 
+  "admin.batch-import.page.header": "Import Batch",
+
   "admin.metadata-import.page.help": "You can drop or browse CSV files that contain batch metadata operations on files here",
 
+  "admin.batch-import.page.help": "You can drop or browse ZIP files that contain batch operations on files here",
```

Comment:
I think we should clarify this description.  You cannot perform "batch operations"... this is just an import.

Perhaps change the description to say:
`Select the Collection to import into. Then, drop or browse to a Simple Archive Format (SAF) zip file which includes the Items you wish to import.`